windows/src/engine/keyman32/keybd_shift.cpp

```diff
@@ -113,15 +113,22 @@ void GetAsyncKeyboardShiftState(BYTE *kbd) {
   }
 }
 
-BOOL keybd_shift(LPINPUT pInputs, int *n, int AShiftFlags, BOOL FPrefix, BOOL FAsync)
+BOOL keybd_shift(LPINPUT pInputs, int *n, BOOL FReset)
```

Comment:
Was able to simplify the parameters to this function. We would be able to refactor even further but I don't want to do that at this time.